Content.Shared/Fluids/Components/DrainComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Fluids.Components;
+
+[RegisterComponent, NetworkedComponent]
+public sealed class DrainComponent : Component
+{
+    public const string SolutionName = "drainBuffer";
+
+    [DataField("accumulator")]
+    public float Accumulator = 0f;
```

Comment:
what's this accumulating?? (doc comment)

Reply:
pretty sure this is just a copy from old code

Reply:
it is yeah I just moved it to shared.

Reply:
This is also why none of it's networked (I just need it in shared for dragdrop to work).